doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc

```diff
@@ -43,6 +43,7 @@ If you do not configure a Git source, the immutable {KIE_SERVER} does not use an
 ====
 +
 .. If you are using S2I and want to set a Git Webhook so that changes in the Git repository cause an automatic rebuild of the {KIE_SERVER}, click *Add new Webhook*. Then select the type of the Webhook in the *Type* field and enter the secret string for the Webhook in the *Secret* field.
+.. If you want to set a build environment variable for the S2I build, click *Add new Build Config Environment variable*, then enter the name and value for the variable in the *Name* and *Value* fields.
```

Comment:
Suggestion - I think there is a preference for using ", and" instead of ", then"